### ISSUE TITLE
Zed editor keybindings and file manager not working

### ISSUE THREAD

**Issue by ArikRahman** [zed-industries/zed#11905]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

zed-editor 0.135.2-1
Open zed
Try keybindings (none work)
Open project panel
Ctrl-O or manually open a project
Nothing appears, no dolphin or anything.
My vscode and chromium open file manager just fine.

### Environment

Fastfetch:
OS:
Archcraft x86_64
Host: Nitro AN515-55 (V2.03)
Kernel: Linux 6.8.9-arch1-2
Uptime: 11 hours, 23 mins
Packages: 1752 (pacman), 7 (flatpak)
Shell: zsh 5.9
Display (NCP004D): 1920x1080 @144Hz [Built-in]
WM: Hyprland (Wayland)
Theme : Catppuccin-Mocha [GTK2], Tokyo-Night [GTK3]
Icons: Tela-circle-dracula [GTK2], Tela-circle-purple [GTK3]
Font: Cantarell (10pt) [GTK2/3]
Cursor: Bibata-Modern-Ice (20px)
Terminal: alacritty 0.13.2
Terminal Font: alacritty (11pt)
CPU: Intel(R) Core(TM) 15-10300H (8) @ 4.50 GHz
GPU 1: Intel UHD Graphics @ 1.05 GHz [Integrated]
GPU 2
: NVIDIA GeForce RTX 3050 Mobile [Discrete]
Memory: 8.71 GiB/15.46 GiB (56%)
Swap: 159.95 MiB / 17.00 GiB (1%)

LSCPU:
Architecture:             x86_64
  CPU op-mode(s):         32-bit, 64-bit
  Address sizes:          39 bits physical, 48 bits virtual
  Byte Order:             Little Endian
CPU(s):                   8
  On-line CPU(s) list:    0-7
Vendor ID:                GenuineIntel
  Model name:             Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz
    CPU family:           6
    Model:                165
    Thread(s) per core:   2
    Core(s) per socket:   4
    Socket(s):            1
    Stepping:             2
    CPU(s) scaling MHz:   20%
    CPU max MHz:          4500.0000
    CPU min MHz:          800.0000
    BogoMIPS:             5001.23
    Flags:                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopol
                          ogy nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3
                          dnowprefetch cpuid_fault epb ssbd ibrs ibpb stibp ibrs_enhanced tpr_shadow flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid mpx rdseed adx smap clflushopt intel_pt xsaveopt xs
                          avec xgetbv1 xsaves dtherm ida arat pln pts hwp hwp_notify hwp_act_window hwp_epp vnmi pku ospke md_clear flush_l1d arch_capabilities
Virtualization features:
  Virtualization:         VT-x
Caches (sum of all):
  L1d:                    128 KiB (4 instances)
  L1i:                    128 KiB (4 instances)
  L2:                     1 MiB (4 instances)
  L3:                     8 MiB (1 instance)
NUMA:
  NUMA node(s):           1
  NUMA node0 CPU(s):      0-7
Vulnerabilities:
  Gather data sampling:   Mitigation; Microcode
  Itlb multihit:          KVM: Mitigation: VMX disabled
  L1tf:                   Not affected
  Mds:                    Not affected
  Meltdown:               Not affected
  Mmio stale data:        Mitigation; Clear CPU buffers; SMT vulnerable
  Reg file data sampling: Not affected
  Retbleed:               Mitigation; Enhanced IBRS
  Spec rstack overflow:   Not affected
  Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl
  Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization
  Spectre v2:             Mitigation; Enhanced / Automatic IBRS; IBPB conditional; RSB filling; PBRSB-eIBRS SW sequence; BHI SW loop, KVM SW loop
  Srbds:                  Mitigation; Microcode
  Tsx async abort:        Not affected

**Comment by ArikRahman** [comment]
Also greeted with auto update failed even with stable internet connection. Image attached of working zed configuration.
![image](https://github.com/zed-industries/zed/assets/40479733/eea5c577-80af-44ab-8e77-837ea30b5133)

**Comment by jansol** [comment]
You need to install an [XDG Desktop Portal that provides a file chooser](https://wiki.archlinux.org/title/XDG_Desktop_Portal#List_of_backends_and_interfaces)

**Comment by ArikRahman** [comment]
   ~ sudo pacman -S xdg-desktop-portal-gtk

[sudo] password for arik:
resolving dependencies...
looking for conflicting packages...

Package (1)                   New Version  Net Change  Download Size

extra/xdg-desktop-portal-gtk  1.15.1-2       0.42 MiB       0.11 MiB

Total Download Size:   0.11 MiB
Total Installed Size:  0.42 MiB

:: Proceed with installation? [Y/n] y
:: Retrieving packages...
 xdg-desktop-portal-gtk-1.15.1-2-x86_64   107.8 KiB   426 KiB/s 00:00 [--------------------------------------] 100%
(1/1) checking keys in keyring                                        [--------------------------------------] 100%
(1/1) checking package integrity                                      [--------------------------------------] 100%
(1/1) loading package files                                           [--------------------------------------] 100%
(1/1) checking for file conflicts                                     [--------------------------------------] 100%
(1/1) checking available disk space                                   [--------------------------------------] 100%
:: Processing package changes...
(1/1) installing xdg-desktop-portal-gtk                               [--------------------------------------] 100%
Optional dependencies for xdg-desktop-portal-gtk
    evince: Print preview
ldconfig: File /usr/lib/libQt5WebEngineWidgets.so.5.15.16 is empty, not checked.
ldconfig: File /usr/lib/libQt5WebEngineWidgets.so is empty, not checked.
ldconfig: File /usr/lib/libQt5WebEngineWidgets.so.5.15 is empty, not checked.
ldconfig: File /usr/lib/libQt5WebEngineWidgets.so.5 is empty, not checked.
:: Running post-transaction hooks...
(1/3) Reloading user manager configuration...
(2/3) Arming ConditionNeedsUpdate...
(3/3) Updating the desktop file MIME type cache...
   ~
I have installed one that would work on a  hyprland system yet it still does not prompt with a file explorer
[xdg-desktop-portal-gtk](https://archlinux.org/packages/?name=xdg-desktop-portal-gtk) 	Yes 	No
I then installed one that would work on a  hyprland system even though it doesn't support file picker and as expected it does not work.
[xdg-desktop-portal-hyprland](https://archlinux.org/packages/?name=xdg-desktop-portal-hyprland)1
